tools/integration_tests/concurrent_operations/setup_test.go

```diff
@@ -62,7 +62,7 @@ func TestMain(m *testing.M) {
 	setup.SetUpTestDirForTestBucketFlag()
 
 	flagsSet := [][]string{
-		{"--kernel-list-cache-ttl-secs=-1"},
+		{"--kernel-list-cache-ttl-secs=-1", "--kernel-list-cache-ttl-secs=0"},
```

Comment:
@Tulsishah is there any expected (significant) increase in runtime of e2e tests with this change? 

If yes, then we should increase the timeout for e2e tests.